web/gtm.js

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable no-param-reassign, prefer-template */
+/* eslint-disable no-param-reassign, prefer-template, @typescript-eslint/naming-convention */
```

Comment:
AFAIK this code is generated by google tag manager and later pasted here, maybe we should just put it in `.eslintignore`?